README.md

```
@@ -10,6 +10,12 @@ Currently, it mainly supports related work for AWS, especially [aws-account-oper
 
 ## Installation
 
+### MacOS
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)